### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Greg0109**
Changelog:
- Added main structure of the HTML website
- Added images, fonts, colors that resemble or are the same as the provided example
- Footer and navigation were cloned properly

Lenguage used:
- HTML
- CSS

Used flexbox for the diagram in the middle and a grid for the suggested articles
Used a media query to detect screen size to change some aspects of the site depending on that (i.e. the footer)

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
## Invalid 🛑 

I can't review the project at the moment, in order to fix this :
* [ ] Pull Request should have a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] fix all your linter errors ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: style.css [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+  width: 95%;
+  margin: auto;
+}
+
+#nyt-logo-container-small {
+  display: flex;
+  justify-content: space-between;
+  width: 95%;
+  margin: auto;
+}
```
Comment (andresporras3423) [inline style.css]: Group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Greg Rábago Sainz & Ryel Banfield** 
Good job with your first submission for this project :muscle:  :muscle:
There are some details to improve before you can go ahead:

- [x] Remove horizontal scrollbar
- [x] Add a screenshot in your README file
- [x] The NY Times logo, in the navigation bar, is a link
- [x] A "Log In" button should appear on the rightmost side of the navbar
- [x] group selectors with the same style definition
- [x] You didn't include the editor's pick section
- [x] You must also use float to position some elements
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1053 @@
...
+              <p class="suggested-title">
+                <a href="#"> Sync your calendar with the solar system </a>
+              </p>
+              <p class="suggested-date">Jan. 5</p>
+            </li>
+            <li>
+              <img src="assets/FooterArticleImages/missionsmars.jpeg" alt="" />
+              <p class="suggested-source">NASA/JPL-Caltech</p>
+              <p class="suggested-title">
+                <a href="">
```
Comment (umairarshadbutt) [inline index.html]: Add a description in `href` attribute

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1053 @@
...
+    />
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <div id="nav-bar-icons">
+          <img
+            src="assets/menu-icon.PNG"
+            alt=""
```
Comment (umairarshadbutt) [inline index.html]: Add a description in `alt` attribute

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1053 @@
...
+            id="menu-icon"
+            class="nav-icon"
+          />
+          <img
+            src="assets/search-icon.PNG"
+            alt=""
```
Comment (umairarshadbutt) [inline index.html]: Add a description in `alt` attribute

**Review by umairarshadbutt — COMMENTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Greg0109 and @RyelBanfield 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes  👇.
- [x] Add a description in `alt` attribute
- [x] Add a description in `href` attribute
- [x] MIT Lisence link is not working in the README file
- [x] Add .gitignore file and remove and remove .DS_Store from repo
- [x] Login button is not visible properly in navabr 
See image
 
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇
### Hi Team✋🏽
Nice work. Your project meets the minimum requirements for it to be approved. It's a thumbs up for me. 👍🏽 You can go ahead and merge this pull request.🤝

Good luck on your next project and Happy Coding 💻

**Comment by Greg0109** [comment]
Thanks!

**Pull request merged**